src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,212 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+class RegistrationServiceImplTest {
+    private static final int NEGATIVE_AGE = -1;
+    private static final int AGE_0 = 0;
+    private static final int AGE_17 = 17;
+    private static final int AGE_18 = 18;
+    private static final int AGE_30 = 30;
+    private static final String FIVE_SYMBOLS = "qwert";
+    private static final String SIX_SYMBOLS = "qwerty";
+    private static final String TEN_SYMBOLS = "?-qwerty@7";
+    private static final String EXISTING_LOGIN = "here_i_am";
+    private static final String NEW_LOGIN = "new_first";
+    private static final String EMPTY_STRING = "";
+    private static final String SPACE_SYMBOL = " ";
+    private static final String TAB_SYMBOL = "  ";
+    private static final String SIX_SPACES = "      ";
```

Comment:
make all these variables local

Reply:
Done. But I cannot understand why we have to make these constants local variables if they are constants? I mean we usually declare them as private static final variables in the class. Moreover, I use some of them not once, but in several methods.

Reply:
You should create constant in tests in this will increase code readability or if you use this variable more than ones in your tests. Lets remove all constant that used only ones

Reply:
Made all constants that used only ones to local variables